### ISSUE TITLE
Capture the Public User Feedback Process

### ISSUE THREAD

**Issue by dshaw** [nodejs/user-feedback#53]
Action item from #46: create a markdown file to capture and document the process we've been using for Public User Feedback sessions.

**Comment by mihaiep** [comment]
I think Michael suggest it on last team meeting we should try [Wiki.](https://github.com/nodejs/user-feedback/wiki). Do we want to create a page- with the header the actual meeting date, and only questions, thoughts, instead of md file? It may look like [this](https://github.com/mihaiep/Test_Area/wiki/2018-04-14-Questions)

**Comment by mhdawson** [comment]
If I remember correctly, in the end I believe we agreed on using a markdown (md) file edited through the github GUI.

**Comment by mihaiep** [comment]
Yes, you're right. Trying to see which way is easier to keep it simple.

**Comment by dshaw** [comment]
Still open and in need of contribution.

**Comment by joesepi** [comment]
What is the goal here and how can I help?

It seems to me that for starters, we need a guide/template for creating and executing User Feedback surveys. Is that the scope expected in this issue?
